src/zrc/api/serializers.py

```diff
@@ -210,6 +230,26 @@ def _get_zaaktype(self, zaaktype_url: str) -> dict:
             self._zaaktype = client.request(zaaktype_url, 'zaaktype')
         return self._zaaktype
 
+    def _get_information_objects(self) -> list:
+        if not hasattr(self, '_information_objects'):
+            self._information_objects = []
+
+            if self.instance:
+                Client = import_string(settings.ZDS_CLIENT_CLASS)
+
+                zios = self.instance.zaakinformatieobject_set.all()
+                for zio in zios:
+                    io_url = zio.informatieobject
+                    client = Client.from_url(io_url)
+                    client.auth = APICredential.get_auth(
+                        io_url,
+                        scopes=['zds.scopes.zaaktypes.lezen']
```

Comment:
should be a DRC scope, not a ZRC scope, right?

Reply:
Ah, true. However, scopes are not yet in DRC, but changed to what it probably will be: `scopes.documenten.lezen`.